javascript/components/FormInput.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class ContactField extends Component {
+  render () {
+    let formElt;
+    const { inputType, inputValue, handleChange, fieldName, options } = this.props;
+    if (inputType === 'dropdown') {
+      formElt = (
+        <select name={this.props.fieldName.toLowerCase().split(' ')[0]} onChange={handleChange}>
```

Comment:
I think it's probably to have the component consumer pass in an explicit `name` prop--especially when we internationalize the field display name.

Reply:
Changed! Thanks.